ride_to_work_by_bike_config.toml

```diff
@@ -2,6 +2,7 @@
 
 colorPrimary = "#43449C"
 colorPrimaryDark = "#110547"
+colorPrimaryOpacity = "#DBDBEB"
```

Comment:
`colorPrimaryOpacity` var value doesn't correspond `colorPrimary` var value and usage of HEX color format is incorrect too.

Correct value should be:

HEX color format #43449C == RGB color format rgb(67, 68, 156)

`colorPrimaryOpacity` var name tell you that you need specify **opacity level** with rgba color format rgba(67, 68, 156, <YOUR_OPACITY_LEVEL>)

```
colorPrimary = "#43449C"
colorPrimaryOpacity = "rgba(67, 68, 156, <YOUR_OPACITY_LEVEL>)"
```

If you need another color definition #DBDBEB you must use another better global var color name.